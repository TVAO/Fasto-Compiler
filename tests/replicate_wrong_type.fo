fun bool writeBool(bool b) = write(b)

fun [bool] main() =
  let fs = replicate("hello", false) in
  map(writeBool, fs)
