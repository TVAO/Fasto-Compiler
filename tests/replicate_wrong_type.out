Type error: Type mismatch in 1st argument of replicate: expected int, but got [char] at line 4, column 12